tests/unit/vector/GramSchmidtTests.hpp

```diff
@@ -183,6 +185,45 @@ namespace ReSolve {
           delete b;
           return status;
         }
-    }; // class
+ 
+        bool verifyAnswer(vector::Vector* x, index_type K,  ReSolve::VectorHandler* handler, std::string memspace, real_type const tol)
+        {
+          ReSolve::memory::MemorySpace ms;
+          if (memspace == "cpu")
+            ms = memory::HOST;
+          else
+            ms = memory::DEVICE;
+
+          vector::Vector* a = new vector::Vector(x->getSize()); 
+          vector::Vector* b = new vector::Vector(x->getSize());
+
+          real_type ip; 
+          bool status = true;
+
+          for (index_type i = 0; i < K; ++i) {
+            for (index_type j = 0; j < K; ++j) {
+              a->update(x->getVectorData(i, ms), ms, memory::HOST);
+              b->update(x->getVectorData(j, ms), ms, memory::HOST);
+              ip = handler->dot(a, b, memory::HOST);
+              if ( (i != j) && !isEqual(ip, 0.0, tol)) {
+                status = false;
+                std::cout << "Vectors " << i << " and " << j << " are not orthogonal!"
+                  << " Inner product computed: " << ip << ", expected: " << 0.0 << "\n";
```

Comment:
Alignment of streaming operators:
```suggestion
                std::cout << "Vectors " << i << " and " << j << " are not orthogonal!"
                          << " Inner product computed: " << ip << ", expected: " << 0.0 << "\n";
```